src/igl/metal/RenderCommandEncoder.mm

```diff
@@ -384,7 +384,14 @@
                 baseInstance:baseInstance];
 #if IGL_PLATFORM_IOS
   } else {
-    [encoder_ drawPrimitives:metalPrimitive_ vertexStart:firstVertex vertexCount:vertexCount];
+    if (!baseInstance) {
+      [encoder_ drawPrimitives:metalPrimitive_
+                   vertexStart:firstVertex
+                   vertexCount:vertexCount
+                 instanceCount:instanceCount];
```

Comment:
`instanceCount` is not supported by all iOS versions. There should be another `if` block.